src/renderer/sw_engine/tvgSwShape.cpp

```diff
@@ -348,13 +348,13 @@ static bool _axisAlignedRect(const SwOutline* outline)
 }
 
 
-static SwOutline* _genOutline(SwShape* shape, const RenderShape* rshape, const Matrix& transform, SwMpool* mpool, unsigned tid, bool hasComposite, bool trimmed = false)
+static SwOutline* _genOutline(SwShape* shape, const RenderShape* rshape, const Matrix& transform, SwMpool* mpool, unsigned tid, bool hasComposite, bool strokeTrim = false)
```

Comment:
`trimmed` looks already good. This is a general path outline. Please do not mix the stroke concept here.

Reply:
but this is really only info about stroke... I'll try to change this after you comment whether we want to support fill only trimming (I asked this in the comment below)